doc/doxygen/scripts/steps.pl

```diff
@@ -26,13 +26,24 @@
   print $line;
 }
 
-# List of additional node attributes to highlight purpose and state of the example
+# List of additional node and edge attributes to highlight purpose and state of
+# a tutorial or code gallery program
+my %colors = (
+ "basic"          => 'green',
+ "techniques"     => 'orange',
+ "fluids"         => 'yellow2',
+ "solids"         => 'lightblue',
+ "time dependent" => 'blue',
```

Comment:
would you mind changing this blue to something lighter? I find it really hard to read black on dark blue.

Reply:
Will do, but let me do that in a separate PR.